presto-orc/src/main/java/com/facebook/presto/orc/metadata/DwrfMetadataReader.java

```diff
@@ -112,6 +118,84 @@ public Footer readFooter(HiveWriterVersion hiveWriterVersion, InputStream inputS
                 footer.hasEncryption() ? Optional.of(toEncryption(footer.getEncryption())) : Optional.empty());
     }
 
+    @Override
+    public List<ColumnStatistics> decryptFileStatistics(HiveWriterVersion hiveWriterVersion,
```

Comment:
This is doing more than decryption.  I would call this decryptAndCombineFileStatistics or something.